components/Common/Button/index.stories.tsx

```diff
@@ -0,0 +1,294 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Button from './index';
+
+const meta = {
+  component: Button,
+  argTypes: {
+    variant: ['primary', 'secondary'],
+  },
+} satisfies Meta<typeof Button>;
+type Story = StoryObj<typeof meta>;
+
+/**
+ * Primary
+ */
+
+export const Primary: Story = {
```

Comment:
These stories (all of them) are violating some of our guidelines. Using the `render` method is not recommended. Please prefer to use `component` and `args`

https://storybook.js.org/docs/react/writing-stories/args